packages/block-library/src/cover/edit.js

```diff
@@ -306,6 +306,8 @@ function CoverEdit( {
 		minHeightUnit,
 		style: styleAttribute,
 		url,
+		allowedBlocks,
+		templateLock = false,
```

Comment:
Setting a default here explicitly removes the locking even if the attribute is unset causing a behavior change. I think we should not include the default here.

Reply:
Same default is in Column PR: https://github.com/WordPress/gutenberg/pull/25778/files 
Maybe there is no problem, but I prefer to inform you.

Reply:
The default value has been removed.